cranelift/docs/heap.dot

```diff
@@ -0,0 +1,8 @@
+digraph {
+        node [
+              shape=record,
+              fontsize=10,
+              fontname="Vera Sans, DejaVu Sans, Liberation Sans, Arial, Helvetica, sans"
+              ]
```

Comment:
As a super minor nit: this `]` should be indented at the level of the `node [`, for consistency with how bracketed constructs are typically formatted in Rust.